core/services/capreg/syncer.go

```diff
@@ -0,0 +1,74 @@
+package capreg
+
+import (
+	"context"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services"
+	"go.uber.org/multierr"
+)
+
+var _ services.ServiceCtx = (*syncer)(nil)
+
+type syncer struct {
+	locals []Local
+	lggr   logger.Logger
+}
+
+// HealthReport implements services.Service.
+func (s *syncer) HealthReport() map[string]error {
+	return map[string]error{} // TODO: implement
+}
+
+// Name implements services.Service.
+func (s *syncer) Name() string {
+	return "CapabilityRegistrySyncer"
+}
+
+// Ready implements services.Service.
+func (s *syncer) Ready() error {
+	return nil // TODO: implement
+}
+
+func NewSyncer(locals []Local, lggr logger.Logger) *syncer {
+	return &syncer{
+		locals: locals,
+		lggr:   lggr.Named("capreg_syncer"),
+	}
+}
+
+// Close implements services.Service.
+func (s *syncer) Close() error {
+	var errs error
+	for _, local := range s.locals {
+		if err := local.Close(); err != nil {
+			errs = multierr.Append(errs, err)
+		}
+	}
+	return errs
+}
+
+// Start implements services.Service.
+func (s *syncer) Start(ctx context.Context) error {
+	tick := time.NewTicker(12 * time.Second)
```

Comment:
Just checking: the way this is currently written, Start will never return (i.e. we're ticking in the main goroutine, not a background one)

Also a gotcha with time.NewTicker is that the first tick happens at T+12s. That may be what you intended but in my experience it usually isn't.